### PR TITLE
Qna file structure and changes to app render structure

### PR DESCRIPTION
**Pull request by davidltruong**
- add files for qna components
- changed app render to include more components
- changed file path for overview from 'widget' to 'component'

### CODE REVIEW

**Comment by zxl3269117** [comment]
@davidltruong Looks good overall. I'll approve after the test suite pull request merge and you should be able to merge the master to this branch and resolve confict (potentailly).

**Review by zxl3269117 — APPROVED** [review]
Looks good.

**Review by mpmanzo — APPROVED** [review]
Approved.

**Pull request merged**
